### PR TITLE
Fix name issue and CI script 

### PR DESCRIPTION
**Pull request by am11**
Now using latest CLI version in CIs.

### CODE REVIEW

**Comment by am11** [comment]
With the latest dotnet-cli, TravisCI was failing when language was set to C++ with "Missing reference assembly" error. Note that the (six-days) older version of CLI isn't failing with that message. So I changed the Travis language from C++ to csharp (which installs mono for each job and yet a slow container.. the reason why I did not had csharp before).

Reference assemblies are only needed on Unix, when we are building against Mono. Our TravisCI script does not enlist Mono in matrix. I cannot reproduce it locally. Also note that CoreFX has no dependency on Mono ref assemblies (it used to, couple of months ago but not anymore). I have not tried reproducing this issue in a fresh VM yet.

File: GeoIP2.UnitTests/project.json [hunk 1]
```diff
@@ -11,7 +11,7 @@
   },
 
   "commands": {
-    "test": "MaxMind.Db.Test"
+    "test": "MaxMind.GeoIP1.UnitTests"
```
Comment (oschwald) [inline GeoIP2.UnitTests/project.json]: The `1` should be a `2`.

**Comment by oschwald** [comment]
This looks and works great for me. Thanks! I had one very minor nit.

File: GeoIP2.UnitTests/project.json (hunk 1 above)
Comment (am11) [inline GeoIP2.UnitTests/project.json]: Hehe, I wonder where that 1 came from..  ¯\ (°_o)/¯ 
Fixed in c0754db. Thanks! :)

**Comment by oschwald** [comment]
Thanks! :+1:

**Comment by oschwald** [comment]
I did a [beta release](https://www.nuget.org/packages/MaxMind.GeoIP2).

**Comment by am11** [comment]
Great! Thank you. 👍

**Pull request merged**